sde-cookbook/sdk/install_sdk.sh

```diff
@@ -11,7 +11,7 @@
 
 SDK_SELF_EXTRACTOR=$1
 PELUX_SDK_DESTINATION="/opt/pelux_sdk"
-PATH_TO_SDK_SELF_EXTRACTOR="${PELUX_SDK_DESTINATION}/${SDK_SELF_EXTRACTOR}"
+PATH_TO_SDK_SELF_EXTRACTOR="${PELUX_SDK_DESTINATION}/`basename ${SDK_SELF_EXTRACTOR}`"
```

Comment:
Why this change?

Reply:
Ah yeah, perhaps I should add a comment explaining this, because we copy the self-extracting-installer from /vagrant/ to /opt/pelux_sdk and we changed that its not directly in /vagrant/ but instead /vagrant/test/stubs/name_sdk.sh, we only need the name of the installer, not the path.

This was necessary because we moved the stub_sdk.sh from root into test/stubs/full_sdk.sh to not pollute the root with all the stub-sdks. It might not be the most elegant solution but we didn't come up with a better one.